src/combinatorics/simple_graph/regularity/equitabilise.lean

```diff
@@ -0,0 +1,237 @@
+/-
+Copyright (c) 2022 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import order.partition.equipartition
+
+/-!
+# Equitabilising a partition
+
+This file allows to blow partitions up into parts of controlled size. Given a partition `P` and
+`a b m : ℕ`, we want to find a partition `Q` with `a` parts of size `m` and `b` parts of size
+`m + 1` such that all parts of `P` are "as close as possible" to unions of parts of `Q`. By
+"as close as possible", we mean that each part of `P` can be written as the union of some parts of
+`Q` along with at most `m` vertices.
```

Comment:
Is `vertices` the best word here? Sounds very graph-theoretic to me. Maybe just `other elements`?

Reply:
This is graph theory, indeed :)

Reply:
The application is graph theory, and I don't know of applications outside of graph theory but the construction itself isn't specific to graphs - so I think Johan's suggestion for this change is fair